sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageBinding.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net;
+using Azure.Storage.Queues;
+using CoreWCF.Channels;
+
+namespace Azure.Storage.CoreWCF.Channels
```

Comment:
We'll need to review namesapces here.   I do not believe that we'd want these to start with `Azure.Storage`.


//cc: @KrzysztofCwalina, @tg-msft